labs/pluggable-hcs

```diff
@@ -0,0 +1,23 @@
+# Lab Name
+_Pluggable Hedera Consensus Service_
+
+# Short Description
+_The Pluggable HCS Lab proves the ability of a Hyperledger Fabric network to connect to the Hedera Consensus Serivce running on a publicly accessilbe testnetwork._
```

Comment:
I suggest using "demonstrates" rather than "proves".

Reply:
just made the change